fractions/jaeger/src/main/java/org/wildfly/swarm/jaeger/JaegerFraction.java

```diff
@@ -0,0 +1,84 @@
+package org.wildfly.swarm.jaeger;
+
+import org.jboss.logging.Logger;
+import org.wildfly.swarm.spi.api.Fraction;
+import org.wildfly.swarm.spi.api.annotations.Configurable;
+import org.wildfly.swarm.spi.api.annotations.DeploymentModule;
+
+import static com.uber.jaeger.Configuration.*;
+
+/**
+ * @author Juraci Paixão Kröhling
+ */
+@DeploymentModule(name = "org.wildfly.swarm.jaeger", slot = "deployment")
+@Configurable("swarm.jaeger")
+public class JaegerFraction implements Fraction<JaegerFraction> {
+    private static final Logger logger = Logger.getLogger(JaegerFraction.class);
+
+    private String serviceName = getDefault(JAEGER_SERVICE_NAME);
```

Comment:
I think it'd be worthwhile looking at Defaultable<String> and our dynamic defaults suppliers.  This would allow YAML to *know* a default value is applied, instead of an implicit default which just happens to be otherwise unset.

https://github.com/wildfly-swarm/wildfly-swarm/blob/master/core/spi/src/main/java/org/wildfly/swarm/spi/api/Defaultable.java#L21